### PR TITLE
Jhalackova adding image types

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Describe the overall goals of this pull request.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

**Comment by lvicoun** [comment]
[Micro-deployment-images_en.pdf](https://github.com/SUSE/doc-modular/files/14635250/Micro-deployment-images_en.pdf)

File: concepts/alp-installation-images.xml [hunk 1]
```diff
@@ -93,7 +69,9 @@
         <listitem>
           <para>
             Instead of installing &productnameshort; in a virtual machine, you can use pre-built
-            disk images for supported hypervisors.
+            disk images for supported hypervisors. The <filename>.qcow</filename> or
+            <filename>.qcow2</filename> are available for the &qemu; emulator. The
+            <filename>VMware</filename> are intended for deployment on the VMware virtualization technology.
```
Comment (dariavladykina) [inline concepts/alp-installation-images.xml:74]:
```suggestion
            disk images for supported hypervisors. The <filename>.qcow</filename> and
            <filename>.qcow2</filename> hypervisors are available for the &qemu; 
            emulator. The <filename>VMware</filename> hypervisors are intended 
            for deployment on the VMware virtualization technology.
```

File: concepts/alp-installation-images.xml [hunk 2]
```diff
@@ -106,11 +84,46 @@
...
       </varlistentry>
     </variablelist>
-    <note>
-      <para>
-        Not all types of installers are available for each installation environment.
-      </para>
-    </note>
+    <para>
+      All of the above mentioned images are delivered as either a <literal>base</literal> or
+      <literal>default</literal> subtype. The subtypes contain besides the core OS itself the following tools:
```
Comment (dariavladykina) [inline concepts/alp-installation-images.xml:89]:
```suggestion
      All the above mentioned images are delivered as either a <literal>base</literal> or
      <literal>default</literal> subtype. In addition to the core OS, subtypes contain
      the following tools:  
```

**Review by dariavladykina — COMMENTED** [review]
Just a couple nits. Thanks!

**Pull request merged**